hyperspy/_components/power_law.py

```diff
@@ -152,3 +155,21 @@ def estimate_parameters(self, signal, x1, x2, only_current=False,
             self.r.map['is_set'][:] = True
             self.fetch_stored_values()
             return True
+
+    def array(self, axis):
+        """%s
+
+        """
+        x = axis[np.newaxis, :]
+        A = self.A.map['values'][..., np.newaxis]
+        r = self.r.map['values'][..., np.newaxis]
+        o = self.origin.map['values'][..., np.newaxis]
+        # Since no parameter can be used to normalise for binned data, we do 
+        # it here.
+        if self.binned:
```

Comment:
Why is multiplying by a factor needed here? ``integrate1D`` in ``estimate_parameters`` should take care of this.